### PR TITLE
 API to count the total no. of files

### PR DESCRIPTION
**Pull request by RaviRajput10**
fixes: #77

### CODE REVIEW

File: apps/api/lib/media/handlers.ts [hunk 1]
```diff
@@ -104,6 +105,17 @@ export async function getMedia(
     }
 }
 
+export async function getMediaCount(req: any, res: any) {
+    try {
+        const userId = req.user._id;
```
Comment (rajat1saxena) [inline apps/api/lib/media/handlers.ts:110]: Move these two lines of out of `try`.

**Review by rajat1saxena — CHANGES_REQUESTED** [review]

File: apps/api/lib/media/handlers.ts (hunk 1 above)
Comment (RaviRajput10) [reply apps/api/lib/media/handlers.ts:110]: Moved up the lines.

**Comment by rajat1saxena** [comment]
If I remove `?page=1` from the URL, the pagination breaks.

Ideally, if the page number is not provided, it is automatically added to the URL. 

image

**Comment by RaviRajput10** [comment]
> If I remove `?page=1` from the URL, the pagination breaks.
> 
> Ideally, if the page number is not provided, it is automatically added to the URL.
> 
> image

**Pull request merged**